index.html

```diff
@@ -274,50 +294,67 @@ <h3>Namespace Specific Identifier (NSI)</h3>
               <code>
-                peer-did      = "did:peer:" keyfmtchar ":" idstring<br>
-                keyfmtchar    = "1"<br>
-                idstring      = 21*22base58char<br>
-                base58char    = "1" / "2" / "3" / "4" / "5" / "6" / "7" / "8" / "9" / "A" / "B" / "C"<br/>
-                  &nbsp;&nbsp;&nbsp;&nbsp;/ "D" / "E" / "F" / "G" / "H" / "J" / "K" / "L" / "M" / "N" / "P" / "Q"<br/>
-                  &nbsp;&nbsp;&nbsp;&nbsp;/ "R" / "S" / "T" / "U" / "V" / "W" / "X" / "Y" / "Z" / "a" / "b" / "c"<br/>
-                  &nbsp;&nbsp;&nbsp;&nbsp;/ "d" / "e" / "f" / "g" / "h" / "i" / "j" / "k" / "m" / "n" / "o" / "p"<br/>
-                  &nbsp;&nbsp;&nbsp;&nbsp;/ "q" / "r" / "s" / "t" / "u" / "v" / "w" / "x" / "y" / "z"<br>
+                peer-did      = "did:peer:" generator "-" idstring<br>
+                generator     = "1"<br>
+                idstring      = multibase multihash<br>
+                multibase     = "F" #see hex in <a target="_blank" href="https://github.com/multiformats/multibase">multibase spec</a><br>
+                multihash     = "1220" restofhash #see sha256+hex in <a target="_blank" href="https://github.com/multiformats/multihash">multihash</a> spec<br>
               </code><br>
-              
-                <p>All Peer DIDs are base58 encoded values. The underlying number represented in base58 is a cryptographic
-                public key or a derivative (e.g., hash) thereof. For future extensibility, the scheme includes a key format 
-                character field (<code>keyfmtchar</code>) that identifies the corresponding <code>idstring</code> to define 
-                the length of the NSI.</p>
-              
-                <p>At the time of this writing, only one <code>keyfmtchar</code> is defined. This is "1", denoting a format 
-                where <code>idstring</code> is the upper 16 bytes of an Ed25519 public key. This gives <code>idstring</code> 
-                a length of either 21 or 22 base58 characters. The encoding of <code>idstring</code> uses most alphas and digits, 
-                omitting 0 (zero), O (capital o), I (capital i), and l (lower L) to avoid readability problems. Regardless of 
-                the length of <code>idstring</code>, peer DIDs are case-sensitive and may not be case-normalized, even though 
-                the prefix is always lower-case.</p>
-              
-                <p>Note that this spec may be updated to include other <code>keyfmtchar</code> variants for
-                SecP256, Curve448, or similar. Such updates might change the length of <code>idstring</code> as well.</p>
+              Peer DIDs use an underlying number with high entropy, called the <em>numeric basis</em>, as the
+                source of their uniqueness. This version of
+                the spec only defines one scheme for generating the numeric basis (<code>generator</code>
```

Comment:
Instead of explicitly saying that the `generator` field specifies how the numeric basis is generated, perhaps this field could be a simple `version` field to make it more flexible how different versions are defined?

Reply:
Okay. I'll update.